atoms/Icon/style.scss

```diff
@@ -1,5 +1,6 @@
 .hui-Icon {
   font-family: FontAwesome;
+  line-height: inherit !important;
```

Comment:
because font-awesome classes get overridden all the time, and we've already run into instances with the widget library where the icon line-height is being overridden because of styles on the 'fa' class.